app/src/main/java/ch/epfl/sdp/blindly/main_screen/audio_player/AudioPlayerFragment.kt

```diff
@@ -51,9 +53,8 @@ class AudioPlayerFragment : Fragment() {
     ): View? {
         // Inflate the layout for this fragment
         val view = inflater.inflate(R.layout.fragment_audio_player, container, false)
-
         val file = File("${context?.filesDir?.absolutePath}/$PRESENTATION_AUDIO_NAME")
-        //TODO Can we obtain the duration from a File?

```

Comment:
sooo... can we get this ? I dont see the duration being obtained from the file

Reply:
I have no idea how since I only have a File... the mediaplayer knows the duration but needs an audioRecord to be instantiated...